server/cmd/code-annotation/handler/assignments.go

```diff
@@ -0,0 +1,74 @@
+package handler
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/go-chi/chi"
+	"github.com/go-chi/render"
+
+	"github.com/src-d/code-annotation/server/cmd/code-annotation/serializer"
+	"github.com/src-d/code-annotation/server/model"
+	"github.com/src-d/code-annotation/server/repository"
+)
+
+func HandleGetAssignmentsForUserExperiment(w http.ResponseWriter, r *http.Request) {
+	experimentID, err := strconv.Atoi(chi.URLParam(r, "experimentId"))
+	if err != nil {
+		render.Render(w, r, serializer.NewErrors(err))
```

Comment:
do you think we can provide better error message here? Currently, it's `strconv.Atoi: parsing "qwe": invalid syntax`. I would suggest add information that we can't parse `experimentId` not a random string.

Reply:
sure thing!
That's because I left as "pending" in the PR desc:
>_improve error handling and error messages_

I just tried to postpone the error messages for the next iteration, so it wouldn't block the API.
If you agree, we'd merge without this "feature".